profiles/package.mask

```diff
@@ -32,6 +32,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Wolfgang E. Sanyer <WolfgangESanyer@Gmail.com> (2021-03-11)
+# Per upstream, this has been deprecated in favour if citeproc. bug #775545
+# opened per the examples - Masked for removal in 30 days
+dev-haskell/pandoc-citeproc
+=sci-mathematics/rstudio-1.3.1093
```

Comment:
What I would've preferred is:
- Add new versions of everything
- Remove old after a while

But I understand that's a mess sometimes with huge things like Haskell, ML, ..., so if we have to, it works I suppose.